react/__snapshots__/examples.spec.jsx.snap

```diff
@@ -8,27 +8,34 @@ exports[`ActionMenu should render examples: ActionMenu 1`] = `
 
 exports[`Avatar should render examples: Avatar 1`] = `
 "<div>
-  <div class=\\"styles__c-avatar___PpDI- styles__c-avatar--medium___hjYh_\\"><svg class=\\"styles__icon___23x3R\\" width=\\"16\\" height=\\"16\\">
+  <div class=\\"styles__c-avatar___PpDI- undefined\\"><svg class=\\"styles__icon___23x3R\\" width=\\"16\\" height=\\"16\\">
       <use xlink:href=\\"#people\\"></use>
     </svg></div>
 </div>"
 `;
 
 exports[`Avatar should render examples: Avatar 2`] = `
 "<div>
-  <div class=\\"styles__c-avatar___PpDI- styles__c-avatar--medium___hjYh_\\" style=\\"color: white;\\"><span class=\\"styles__c-avatar-initials___310qC\\">CD</span></div>
+  <div class=\\"styles__c-avatar___PpDI- undefined\\" style=\\"color: white;\\"><span class=\\"styles__c-avatar-initials___310qC\\">CD</span></div>
 </div>"
 `;
 
 exports[`Avatar should render examples: Avatar 3`] = `
 "<div>
-  <div class=\\"styles__c-avatar___PpDI- styles__c-avatar--medium___hjYh_\\"><img src=\\"https://cozy.io/fr/images/cozy-logo_white.png\\" class=\\"styles__c-avatar-image___3yJnb\\" alt=\\"\\"></div>
+  <div class=\\"styles__c-avatar___PpDI- undefined\\"><img src=\\"https://cozy.io/fr/images/cozy-logo_white.png\\" class=\\"styles__c-avatar-image___3yJnb\\" alt=\\"\\"></div>
```

Comment:
do we really want `undefined` as a valid classname ?

Reply:
Isn't CSS-Modules or webpack or whatever supposed to ignore called classnames that don't exist? That's what it used to happen.

Reply:
I don't know but I'm pretty sure that I want to see `styles__c-avatar--medium___hjYh_` in my HTML instead of `undefined` or even nothing. No?

Reply:
Should be better.

Reply:
It looks ok now. Shall we merge ?